src/config.ts

```diff
@@ -274,6 +274,29 @@ export const ARNS_RESOLVER_PRIORITY_ORDER = env
   .varOrDefault('ARNS_RESOLVER_PRIORITY_ORDER', 'on-demand,gateway')
   .split(',');
 
+export const ARNS_ON_DEMAND_CIRCUIT_BREAKER_TIMEOUT_MS = +env.varOrDefault(
+  'ARNS_ON_DEMAND_CIRCUIT_BREAKER_TIMEOUT_MS',
+  `${60 * 1000}`, // 1 minute
```

Comment:
1 minute seems a bit long. How long does the CU typically take to respond?

Reply:
for this process, should be more like ~3 seconds